### PR TITLE
feat(indexer): Add api to docker-compose

### PR DESCRIPTION
**Pull request by roninjin10**
- add API to docker-compose
- This is still a WIP:w

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5942** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5942" target="_blank">Graphite</a> 
    * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
      * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
        * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
          * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
            * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
              * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a> 
                * **PR #5950** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5950" target="_blank">Graphite</a> 
                  * **PR #5951** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951" target="_blank">Graphite</a>  👈
                    * **PR #5952** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951?utm_source=stack-comment).

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5942** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5942" target="_blank">Graphite</a> 
    * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
      * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
        * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
          * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
            * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
              * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a> 
                * **PR #5950** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5950" target="_blank">Graphite</a> 
                  * **PR #5951** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951" target="_blank">Graphite</a>  👈
                    * **PR #5952** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951?utm_source=stack-comment).

File: indexer/docker-compose.yml [hunk 1]
```diff
@@ -16,33 +16,24 @@ services:
...
-      - INDEXER_L1_ADDRESS_MANAGER_ADDRESS=${INDEXER_L1_ADDRESS_MANAGER_ADDRESS:-0xdE1FCfB0851916CA5101820A69b13a4E276bd81F}
     ports:
       - 8080:8080
+    environment:
+      - INDEXER_DB_PORT=5432
+      - INDEXER_DB_USER=db_username
+      - INDEXER_DB_PASSWORD=db_password
+      - INDEXER_DB_NAME=db_name
+      - INDEXER_DB_HOST=postgres
+      - INDEXER_CONFIG=/configs/indexer.toml
```
Comment (hamdiallam) [inline indexer/docker-compose.yml:34]: what part of the indexer config does the api need? Do you think the api module will need access access the rpc urls?

**Comment by roninjin10** [comment]
Will do this from scratch

**Pull request closed without merging**